optics/src/main/scala/io/circe/optics/JsonObjectOptics.scala

```diff
@@ -22,7 +22,7 @@ trait JsonObjectOptics extends CatsConversions with ListInstances {
     }
   }
 
-  implicit lazy val objectAt: At[JsonObject, String, Json] = new At[JsonObject, String, Json]{
+  implicit lazy val objectAt: At[JsonObject, String, Option[Json]] = new At[JsonObject, String, Option[Json]]{
```

Comment:
I think the Scalastyle line-length checker will complain here—it wants lines to be shorted than 100 characters.